### PR TITLE
Fix json parsing routine

### PR DESCRIPTION
**Pull request by CXZ7720**
Change json parsing source from ONCHigh to DrugBank

### CODE REVIEW

File: app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt [hunk 1]
```diff
@@ -110,9 +117,14 @@ class InteractionResultActivity : AppCompatActivity() {
...
 
     }
-    fun changeResultText(result: String?) {
+    fun changeResultText(result: MutableList<String>) {
         loadingCircle.visibility = View.GONE
-        resultTest.text = result
+        var text = ""
+        for (i in result){
+            text += i
+            text += "\n"
```
Comment (Taewan-P) [inline app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt:125]: Can we have more rich response? 
For example, like a bulleted list
* This medicine interacts with that medicine
* This ingredient will dissolve this medicine

**Review by Taewan-P — COMMENTED** [review]
Is this applicable?

File: app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt (hunk 1 above)
Comment (Taewan-P) [reply app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt:125]: I think dash is okay too.
```
- Lorem ipsum dolor sit amet, consectetur adipiscing elit
- sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment (CXZ7720) [reply app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt:125]: I think Bulleted list is nice idea. I'll add it to the next commit.

**Comment by CXZ7720** [comment]
KakaoTalk_20200323_191715452_01
I just committed about bullet list.

**Pull request merged**